core/src/apps/cardano/sign_tx.py

```diff
@@ -472,12 +472,18 @@ async def _show_standard_tx(
     is_network_id_verifiable = _is_network_id_verifiable(msg)
 
     if not is_network_id_verifiable:
-        await show_warning_tx_network_unverifiable(ctx)
+        await fail_if_strict(
```

Comment:
I've also added `fail_if_strict` here, since I assume we don't want to allow empty outputs for non-advanced users? Or is there some standard use case for this?

Reply:
yes, registering staking keys when an input covers the fees+deposit exactly is a legitimate usecase which IMHO makes sense even in strict mode